src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -604,7 +575,11 @@ class LSFJobManager extends BatchEuphoriaJobManagerAdapter {
 
     @Override
     String parseJobID(String commandOutput) {
-        return commandOutput
+        String result = commandOutput.find("<[0-9]+>")
```

Comment:
How can LSF ids look like? Could you put in some examples?

12354.something? or just 1234?

Reply:
Adapting the pattern. Would that not mean leaking Roddy details into BE?

The pattern, which usually works, shows that LSF IDs are just numbers.